MimeMapping.csproj

```diff
@@ -4,11 +4,15 @@
     <TargetFrameworks>netstandard1.1;netstandard2.0;netstandard2.1</TargetFrameworks>
     <Description>Constants for (almost) all MIME types and method to determine MIME type from a file name. Contains just over 1000 mime types.
 
-Generated from the Apache server mime.types file and the H5PB nginx mime.types file. Works similar to .NET's System.Web.MimeMapping.GetMimeMapping.
+Generated from the jshttp/mime-db dp.json file. Works similar to .NET's System.Web.MimeMapping.GetMimeMapping.
```

Comment:
Can you update this repo's readme with a similar change?

Reply:
link is below in `See` section